http://code.google.com/p/darkice/issues/detail?id=29
SerialUlaw.cpp:296:26: error: ‘perror’ was not declared in this scope

--- src/SerialUlaw.cpp
+++ src/SerialUlaw.cpp
@@ -40,6 +40,8 @@
 #include "config.h"
 #endif
 
+#include <stdio.h>
+
 #ifdef HAVE_UNISTD_H
 #include <unistd.h>
 #else
